src/export_globals.rs

```diff
@@ -0,0 +1,123 @@
+use parity_wasm::elements;
+
+use crate::optimizer::{global_section, export_section};
+
+/// Export all declared mutable globals.
+pub fn export_mutable_globals(
+	module: &mut elements::Module,
+	prefix: impl Into<String>,
+) {
+
+	let exports = global_section(module).map(
+		|section| section.entries().iter().enumerate().filter_map(
+			|(index, global)| if global.global_type().is_mutable() { Some(index) } else { None }
+		).collect::<Vec<_>>()
+	).unwrap_or_default();
+
+	if module.export_section().is_none() {
+		module.sections_mut().push(elements::Section::Export(elements::ExportSection::default()));
+	}
+
+	let mut symbol_index = 0usize;
+	let prefix: String = prefix.into();
+	for export in exports {
```

Comment:
Nit: aren't those a vector of mutable globals? May be `mut_globals` would be a more appropriate?